src/users/users.service.ts

```
@@ -1612,4 +1638,309 @@ export class UsersService {
+      await this.prismaService.userOAuthConnection.update({
+        where: { id: existingConnection.id },
+        data: {
+          rawProfile: userInfo as any,
+          updatedAt: new Date(),
+        },
+      });
+
+      return [
+        await this.getOAuthUserDtoById(
+          existingConnection.user.id,
+          existingConnection.user.id,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)